### ISSUE TITLE
Choose and implement a new colour palette

### ISSUE THREAD

**Issue by rossng** [clowdr-app/clowdr#220]
We have had a few goes at this already, but picking colours is suprisingly difficult. However, we would like to have a nice colour palette that reflects a consistent Clowdr brand identity.

Paging @bcpierce00 @EdNutting

**Comment by bcpierce00** [comment]
Happy to help with the palette, once someone tells me the right way to set
the color of a thing!  :-)

**Comment by rossng** [comment]
Ed and I spent some hours experimenting with this a few weeks back. Our conclusion was that the best way to do things (technically speaking) was to:

* Define a new colour palette in [Chakra's expected format](https://chakra-ui.com/docs/theming/theme#colors)
  * Note: we're not actually constrained by the exact selection of names or shades that the default Chakra/Tailwind palette has, but I've found that their system works quite well
  * The Chakra docs suggest a few colour palette generation tools. If you Google 'tailwind palette generator' you will find a bunch more.
* Choose which colours are our 'main/brand' and 'accent' colours, then start using those colours for most of our UI elements. The other colours in the palette can still be used where appropriate, but much less frequently.
